### DISCUSSION TITLE
Please update README about iOS TestFlight not working

### DISCUSSION THREAD

**Discussion by DGCK81LNN** [ppy/osu#16299] [Q&A] [answered]
I understand that [iOS builds are failing](https://twitter.com/ppy/status/1402327785348091904?s=20) but please update the README in case anyone gets confused

**Comment by peppy** [comment, marked as answer]
They are not failing.

**Reply by DGCK81LNN** [reply]
umm, so the testflight got back to normal a long time ago? apologies for posting nonsense :cry:

**Reply by bdach** [reply]
yeah. see https://github.com/ppy/osu/issues/15742.
